nodejs/extensions.bzl

```diff
@@ -2,11 +2,20 @@
 
 load(":repositories.bzl", "DEFAULT_NODE_VERSION", "nodejs_register_toolchains")
 
+DEFAULT_NODE_REPOSITORY = "nodejs"
```

Comment:
sorry, another nit, maybe this should live next to DEFAULT_NODE_VERSION?

Reply:
Updated.